charts/generic-service/templates/controller.yaml

```diff
@@ -16,6 +16,13 @@ metadata:
   {{- if .Values.annotations }}
     {{- .Values.annotations | toYaml | nindent 4 }}
   {{- end }}
+  {{- with .Values.rollout.notifications }}
+    {{- range $event, $channels := . }}
+    {{- range $channel := $channels }}
+    notifications.argoproj.io/subscribe.{{ $event }}.slack: {{ join ";" $channels | quote }}
```

Comment:
I'd prefer to keep this more generalized, since not everybody uses Slack for notifications. Perhaps we could just use the existing `annotations` for this?

Reply:
Yeah that's what I initially wanted to do but it would add the annotation to all resources while it would be only needed for `Rollout`?

Reply:
generic-service applies `labels` to all resources but `annotations` only to the resource created in `controller.yaml`, `Rollout` in this case.

Reply:
removed it 👍